cloud/scope/powervs_machine.go

```diff
@@ -635,7 +617,7 @@ func (m *PowerVSMachineScope) GetRawBootstrapData() ([]byte, error) {
 	secret := &corev1.Secret{}
 	key := types.NamespacedName{Namespace: m.Machine.Namespace, Name: *m.Machine.Spec.Bootstrap.DataSecretName}
 	if err := m.Client.Get(context.TODO(), key, secret); err != nil {
-		return nil, fmt.Errorf("failed to retrieve bootstrap data secret for IBMPowerVSMachine %s/%s: %w", m.Machine.Namespace, m.Machine.Name, err)
```

Comment:
Any reason to remove this info?

Reply:
We are embedding the machine names, namespaces, cluster name so on into logger and thats added to context, Like
```
log = log.WithValues("Machine", klog.KObj(machine))
ctx = ctrl.LoggerInto(ctx, log)
```
These can be found in function
```func (r *IBMPowerVSMachineReconciler) Reconcile(ctx context.Context, req ctrl.Request) (_ ctrl.Result, reterr error) {```

So the log line will by default contains these information and we need not explicitly add it again.